compiler-cli/src/dependencies.rs

```diff
@@ -277,7 +277,7 @@ fn read_manifest_from_disc(paths: &ProjectPaths) -> Result<Manifest> {
 
 fn write_manifest_to_disc(paths: &ProjectPaths, manifest: &Manifest) -> Result<()> {
     let path = paths.manifest();
-    fs::write(&path, &manifest.to_toml())
+    fs::write(&path, &manifest.to_toml(&get_current_directory()?))
```

Comment:
We can't use the current directory as there's no guarantee that the root of the project will be in the current directory.

The `ProjectPaths` is what to use here as it is positioned at the root of the project.